src/material/datepicker/date-range-input.html

```diff
@@ -2,19 +2,21 @@
   class="mat-date-range-input-container"
   cdkMonitorSubtreeFocus
   (cdkFocusChange)="_updateFocus($event)">
-  <div class="mat-date-range-input-start-wrapper">
+  <label class="mat-date-range-input-start-wrapper">
```

Comment:
Should the label have a `for` attribute pointing to the ID of the input that is being projected?

Reply:
It doesn't need the `for` here because the `<input>` is nested inside the `<label>`.